packages/libs/eda/src/lib/map/analysis/appState.ts

```diff
@@ -28,12 +29,35 @@ export const MarkerConfiguration = t.intersection([
   t.union([
     t.type({
       type: t.literal('barplot'),
-      selectedValues: t.union([BinDefinitions, t.undefined]), // user-specified selection
+      selectedValues: t.union([BinDefinitions, t.array(t.string), t.undefined]), // user-specified selection
```

Comment:
I'm not quite sure what's going on here. Doesn't the `pie` version below also need `BinDefinitions`?  Or maybe neither need it (nope - tried that and it doesn't compile).  I think I wrote the original types, so I should know. Need lunch though, so going to hit "send"!

Reply:
Well, I still don't get it, but I'll have to move on.

Both barplot and pie markers can display continuous variables, and so both should need to be able to store the `BinDefinitions` in the config. Or at least they would if completely custom user-defined bins were allowed (no plans for this, don't worry).

But yeah, right now the `markerConfigurations` are not storing `selectedValues` for continuous variables. See what I grabbed from dev tools below.  Both pie and barchart configurations have `undefined` `selectedValues`:

```
[
  {
    "type": "pie",
    "selectedVariable": {
      "entityId": "EUPATH_0043226",
      "variableId": "EUPATH_0030018"
    },
    "selectedCountsOption": "filtered",
    "binningMethod": "quantile"
  },
  {
    "type": "barplot",
    "selectedPlotMode": "count",
    "selectedVariable": {
      "entityId": "EUPATH_0043226",
      "variableId": "EUPATH_0030017"
    },
    "dependentAxisLogScale": true,
    "selectedCountsOption": "filtered",
    "binningMethod": "quantile"
  }
]
```

Reply:
So I'm pretty sure the `BinDefinitions` shouldn't be needed in line 32 above, but will require some other changes to make it compile.